_data/lang/fr.yml

```diff
@@ -0,0 +1,263 @@
+# Mr. Green Jekyll Theme (https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme)
```

Comment:
After rebasing with main check the unused code, check it from the spanish or english version